kie-drools-wb-parent/kie-drools-wb-webapp/pom.xml

```diff
@@ -1950,6 +1950,35 @@
         </configuration>
       </plugin>
       <plugin>
+        <groupId>org.wildfly.swarm</groupId>
+        <artifactId>wildfly-swarm-plugin</artifactId>
+        <configuration>
+          <fractionDetectMode>never</fractionDetectMode>
+          <fractions>
+            <fraction>logging</fraction>
+            <fraction>undertow</fraction>
+            <fraction>cdi</fraction>
+            <fraction>ejb</fraction>
+            <fraction>naming</fraction>
+            <fraction>remoting</fraction>
+            <fraction>jaxrs</fraction>
+            <fraction>ee</fraction>
+            <fraction>security</fraction>
+            <fraction>management</fraction>
+            <fraction>msc</fraction>
+            <fraction>datasources</fraction>
```

Comment:
I will try to remove some these fractions. In theory at least`msc` and `datasources` should be needed.